client/blocks/plan-storage/bar.jsx

```diff
@@ -58,7 +58,7 @@ class PlanStorageBar extends Component {
 		return (
 			<div className={ classes }>
 				<ProgressBar
-					className="plan-storage__bar"
+					className="plan-storage__storage-bar"
```

Comment:
Yeah, doesn't seem necessary might as well drop it (7a0852b2f7eabe1806d00c1af1c29ef6cc81e8e0).

Thanks both!